assets/css/NewEvaluation.less

```diff
@@ -96,7 +95,7 @@ li {
 }
 }
 
-.evalForms-title {
+.eval-forms-title {
```

Comment:
I actually preferred the first one, having a different purpose for every separator made sense. `evalForms` was an entity and `title` was another. Now you don't know what the purpose of the separator is anymore.